app/screens/mypage/ChangeProfileScreen.tsx

```diff
@@ -0,0 +1,169 @@
+  require('../../../assets/images/character3.png'),
+];
+
+type FormData = {
+  nickname: string;
+  avatarId: number;
+};
+type Error = Partial<FormData>;
+
+type ChangeProfileScreenProp = StackScreenProps<RootStackParams>;
+const ChangeProfileScreen: React.FC<ChangeProfileScreenProp> = ({
+  navigation,
+}) => {
+  const toast = useToast();
+  const [formData, setFormData] = React.useState<FormData>({
+    nickname: '',
+    avatarId: 1,
+  });
+  const [errors, setErrors] = React.useState<Error>({});
+  const validate = () => {
+    if (formData.nickname === '') {
+      setErrors({
+        ...errors,
+        nickname: '닉네임을 작성해주세요',
+      });
+      return false;
+    } else {
+      delete errors.nickname;
+    }
+    return true;
+  };
+
+  const { mutate } = useSWRConfig();
+  const registerUser = () => {
+    const isValidate = validate();
+    if (isValidate === false) return;
+    callAPI('/user/username', 'POST', {
+      username: formData.nickname,
+    })
+      .then((res) => res.json())
+      .then((res) => {
+        if (res.isNameExist) {
+          toast.show({
+            status: 'error',
+            title: '닉네임 중복',
+            description: '중복된 닉네임이 존재합니다.',
+          });
+          return;
+        } else {
+          callAPI('/user', 'PUT', {
+            username: formData.nickname,
+            avatar: formData.avatarId.toString(),
+          })
+            .then(() => {
+              toast.show({
+                status: 'success',
+                title: '프로필 수정 성공',
+                description: '프로필 수정이 성공적으로 완료되었습니다.',
+              });
+              mutate('/me');
+              navigation.goBack();
+            })
+            .catch((err) => {
+              console.error(err);
+              toast.show({
+                status: 'error',
+                title: '프로필 수정 실패',
+                description: '프로필 수정 도중 문제가 발생했습니다.',
+              });
+            });
+        }
+      })
+      .catch((err) => {
+        console.error(err);
+        toast.show({
+          status: 'error',
+          title: '프로필 수정 실패',
+          description: '프로필 수정 도중 문제가 발생했습니다.',
+        });
+      });
+  };
+
+  return (
+    <Layout>
+      <VStack space={5}>
+        <FormControl isRequired isInvalid={'nickname' in errors}>
+          <FormControl.Label
+            mb="4"
+            _text={{ fontSize: 'md', fontWeight: 'bold' }}
+          >
+            닉네임
+          </FormControl.Label>
+          <Input
+            value={formData.nickname}
+            variant="filled"
+            placeholder="닉네임을 10자 이내로 입력해주세요"
+            maxLength={10}
+            onChangeText={(value: string) =>
+              setFormData({ ...formData, nickname: value })
+            }
+          />
+          {'nickname' in errors && (
+            <FormControl.ErrorMessage
+              _text={{ fontSize: 'xs', color: 'error.500', fontWeight: 500 }}
+            >
+              {errors['nickname']}
```

Comment:
요거 아래와 같이 바꾸고 파일 맨 위의 `@typescript-eslint/dot-notation` 주석 지우면 좋을 것 같아요~
```suggestion
              {errors.nickname}
```

Reply:
[61f62cf](https://github.com/mountain-sailors/stock-battle-front/pull/65/commits/61f62cf608d5c167e810c40125167542ae087092) 에 수정완료했습니다~